sdv/sampling/hierarchical_sampler.py

```diff
@@ -123,7 +117,33 @@ def _add_child_rows(self, child_name, parent_name, parent_row, sampled_data, num
                 sampled_data[child_name] = pd.concat(
                     [previous, sampled_rows]).reset_index(drop=True)
 
-    def _sample_children(self, table_name, sampled_data):
+    def _enforce_table_sizes(self, child_name, table_name, scale, sampled_data):
+        total_num_rows = int(self._table_sizes[child_name] * scale)
+        for foreign_key in self.metadata._get_foreign_keys(table_name, child_name):
+            num_rows_key = f'__{child_name}__{foreign_key}__num_rows'
+            min_rows = self._min_child_rows[num_rows_key]
```

Comment:
We'll need to change this to `getattr` and provide a sensible default or alternate codepath for backwards compatibility.